### DISCUSSION TITLE
Extreme Environements

### DISCUSSION THREAD

**Discussion by SpaceArchitecture** [NREL/EnergyPlus#8972] [Ideas]
Hello all, quick question regarding extreme environments.

Are there any plans to allow the .epw format to include extreme environments such as Mars or the Moon, with very low pressure and almost no humidity ?

**Comment by Myoldmopar** [comment]
I've actually had thoughts about this and plan to investigate things like this.  The problem is that the EPW change alone would not even close to sufficiently affect the simulation results to make them accurate in other planet conditions.  There will need to be drastic changes to the physics calculations, modifications to solar calculations, the meaning of a "day", among other things.  All that said, EnergyPlus is already capable of simulating moving and rotating buildings with water boundary conditions (ships), so this could be a very interesting thing for _someone_ to investigate. :woman_astronaut:

**Reply by SpaceArchitecture** [reply]
Indeed, if you need more "someone" to help I would be glad to give your project my time, I have a master in Space Architecture from the University of Houston and I have been using Ladybug and Honeybee to write a paper on the automated creation of a base on Mars, it's just a proof of concept for now but there could be some follow-up papers....
